source/docs/useful-resources.rst

```diff
@@ -55,24 +53,20 @@ Strategy
 CAD
 ---
 
-`Autodesk Education Account <https://www.autodesk.com/education/edu-software/overview>`_ --- Autodesk's education account sign up. An education account gives access to both Fusion 360 and Inventor.
-
-`Creo Education License <https://www.ptc.com/en/products/education/free-software/standalone-educator>`_ --- Creo's education license application form.
-
 `Onshape Education Account <https://www.onshape.com/en/education/>`_ --- Onshape's education account creation page.
 
+`Autodesk Education Account <https://www.autodesk.com/education/edu-software/overview>`_ --- Autodesk's education account sign up. An education account gives access to both Fusion 360 and Inventor. Note that education accounts are only available to students of "qualified educational institutions", and you will need to upload a form of documentation (School transcript, School-issued confirmation letter, or School ID) to verify your account.
+
 `SolidWorks Sponsorship <https://app.smartsheet.com/b/form/6762f6652a04487ca9786fcb06b84cb5>`_ --- A link to the application form for free Solidworks licenses for robotics teams.
 
+`Creo Education License <https://www.ptc.com/en/products/education/free-software/standalone-educator>`_ --- Creo's education license application form.
+
 Part Libraries
 ^^^^^^^^^^^^^^
 
-`10650 Hazmat Robotics Public CAD Library <https://workbench.grabcad.com/workbench/projects/gcpgZgLBwhIdL0FfUKJJfM75cqa9RW1ncXaL-lQ4KOl1wa#/space/gcSzacmSeI-l19BYQNPm422pSHLenRxOxVtmaD-Pzynwsq/folder/6578524>`_ --- A multi-vendor part library that works across CAD software due to its use of STEP files.
-
-`2901 Purple Gears Onshape Parts Library <https://ftconshape.com/introduction-to-the-ftc-parts-library/>`_ --- A multi-vendor parts library specifically for Onshape.
-
-`REV Robotics Official CAD Library <https://workbench.grabcad.com/workbench/projects/gcEvgrMnw6kRPx7OR6r45Gvb2t-iOdLiNG3m_ALpdGYzK_#/space/gcFd6nwp5Brrc3ks-92gagLZCV2FkceNTX3qGzaMvy2wQD/folder/2906404>`_ --- REV's official CAD library for its FTC parts.
+`Onshape Parts Library <https://ftconshape.com/introduction-to-the-ftc-parts-library/>`_ --- A multi-vendor parts library specifically for Onshape, originally started by team 2901 Purple Gears, and maintained by the community.
 
-`ServoCity Official CAD Library <https://www.servocity.com/step-files>`_ --- ServoCity's entire STEP file catalog. This includes Actobotics parts.
+.. note:: This section used to have many more libraries, but all of the other libraries that were previously listed here no longer exist.
```

Comment:
This does not need to be here; we do not need to encode information about something that is no longer there. If someone is interested in that, that is what git history is for :).